snc.sh

```diff
@@ -251,12 +251,6 @@ ${OC} delete statefulset,deployment,daemonset --all -n openshift-insights
 # Scale route deployment from 2 to 1
 ${OC} patch --patch='{"spec": {"replicas": 1}}' --type=merge ingresscontroller/default -n openshift-ingress-operator
 
-# Scale console deployment from 2 to 1
-${OC} scale --replicas=1 deployment.apps/console -n openshift-console
-
-# Scale console download deployment from 2 to 0
-${OC} scale --replicas=0 deployment.apps/downloads -n openshift-console
-
```

Comment:
How many console deployments/pods/... will we get after this change? Does this have an adverse impact on our cluster memory footprint?

Reply:
@cfergeau no the memory footprint is almost same as we have previously.